### PR TITLE
Proof of Concept Commit

### PR DESCRIPTION
**Pull request by k-sashank**
Pull Request for all documents till the Proof of Concept

### CODE REVIEW

File: Proof of Concept/Spark_POC.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
+{
```
Comment (oindrillac) [inline Proof of Concept/Spark_POC.ipynb:1]: <p>Great work team! Can you run the Baton detection cells and push it? Have you evaluated how well this is doing on your videos? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-nlgma-body-cam/pull/2/discussion/#comment-1392856187'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by trgardos** [comment]
Just a procedural question and comment. This PR is called PoC, but I see files from EDA and research. This seems to be all the changes so far, not just PoC. I can show you how to make this cleaner.

File: Proof of Concept/Spark_POC.ipynb (hunk 1 above)
Comment (trgardos) [inline Proof of Concept/Spark_POC.ipynb:1]: <p>Nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-nlgma-body-cam/pull/2/discussion/#comment-1392925776'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (trgardos) [inline Proof of Concept/Spark_POC.ipynb:1]: <p>Interesting!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-nlgma-body-cam/pull/2/discussion/#comment-1392925781'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (trgardos) [inline Proof of Concept/Spark_POC.ipynb:1]: <p>Who did the fine tuning? You? Very nice if so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-nlgma-body-cam/pull/2/discussion/#comment-1392925786'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by trgardos — APPROVED** [review]
Approving. In the future it would be good to have a cleaner PR. I can show you how if needed.

**Comment by k-sashank** [comment]
proof of concept

**Review by oindrillac — APPROVED** [review]
Thanks, approving. Please open new PRs for the next steps for the final deployment.

**Pull request merged**